### PR TITLE
Update elastic-agent-command-line.asciidoc

### PR DESCRIPTION
**Pull request by mostlyjason**
Upgrade command should only be be used in standalone mode

### CODE REVIEW

**Review by EricDavisX — APPROVED** [review]
confirmed conversation with Engineering - this is the spec.  approved, thanks Jason

File: docs/en/ingest-management/elastic-agent/elastic-agent-command-line.asciidoc [hunk 1]
```diff
@@ -383,7 +383,7 @@ elastic-agent uninstall
 [[elastic-agent-upgrade-command]]
 == elastic-agent upgrade
 
-Upgrade the currently running {agent} to the specified version.
+Upgrade the currently running {agent} to the specified version. This should only be used with agents running in standalone mode. Agents enrolled in Fleet should be upgraded through Fleet.
```
Comment (dedemorton) [inline docs/en/ingest-management/elastic-agent/elastic-agent-command-line.asciidoc:386]:
```suggestion
Upgrade the currently running {agent} to the specified version. This should only be used with agents running in standalone mode. Agents enrolled in {fleet} should be upgraded through {fleet}.
```

**Review by dedemorton — COMMENTED** [review]
I've suggested a minor change to use attributes instead of hard coding Fleet (we try to clean these up when we can).

Thanks for adding this! It will need to be backported to the master and 7.x branches. Let me know if you want me to do that.

**Comment by dedemorton** [comment]
I'll go ahead and merge/backport this unless you say no. :-)

**Comment by dedemorton** [comment]
I forgot to merge/backport. Doing that now. :-)

**Pull request merged**